packages/react-instantsearch-core-next/src/SearchIndex.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Why call this file `SearchIndex` and return `Index`? Do we envision other kind of indices (e.g., `RecommendIndex`)?

If yes, I'd rename the exported component. If not, i'd rename the file.

Reply:
Otherwise we get `index.ts` and `Index.tsx` in the same folder and it quickly gets confusing for the tooling. In fact, it triggers an error.

It's only the file name though, the component remains `Index`.

Another solution is not to use `index.ts` in the same folder. But the file architecture might change so I think we can talk about it later on.

Reply:
No that makes sense to keep it like that indeed, I didn't think of filename case-sensitivity.